README.md

```diff
@@ -232,15 +233,15 @@ Models from https://github.com/onnx/models that have been tested to work with th
 
 
 ### Operators
-List of [ONNX operators supported in CoreML 2.0 via the converter](https://github.com/onnx/onnx-coreml/blob/4d8b1cc348e2d6a983a6d38bb6921b6b77b47e76/onnx_coreml/_operators.py#L1893)
+List of [ONNX operators supported in Core ML 2.0 via the converter](https://github.com/onnx/onnx-coreml/blob/4d8b1cc348e2d6a983a6d38bb6921b6b77b47e76/onnx_coreml/_operators.py#L1893)
 
-List of [ONNX operators supported in CoreML 3.0 via the converter](https://github.com/onnx/onnx-coreml/blob/4d8b1cc348e2d6a983a6d38bb6921b6b77b47e76/onnx_coreml/_operators_nd.py#L1821)
+List of [ONNX operators supported in Core ML 3.0 via the converter](https://github.com/onnx/onnx-coreml/blob/4d8b1cc348e2d6a983a6d38bb6921b6b77b47e76/onnx_coreml/_operators_nd.py#L1821)
 
 Some of the operators are partially compatible with Core ML, for example gemm with more than 1 non constant input is not supported in Core ML 2, or scale as an input for upsample layer is not supported in Core ML 3 etc.
-For unsupported ops or unsupported attributes within supported ops, CoreML custom layers or custom functions can be used.   
-See the testing script `tests/custom_layers_test.py` on how to produce CoreML models with custom layers and custom functions. 
+For unsupported ops or unsupported attributes within supported ops, Core ML custom layers or custom functions can be used.
+See the testing script `tests/custom_layers_test.py` on how to produce Core ML models with custom layers and custom functions.
 
 ## License
-Copyright © 2018 by Apple Inc., Facebook Inc., and Prisma Labs Inc.
+Copyright © 2018-2019 by Apple Inc., Facebook Inc., and Prisma Labs Inc.
```

Comment:
Do Facebook and Prisma owns Copyright 2018 onwards?

Reply:
Leave it unchanged for now.